backend/services/WorkflowEngine.js

```
@@ -0,0 +1,589 @@
+  async handleWorkflowAction(data) {
+    const { action, payload } = data;
+
+    switch (action) {
+      case 'create':
+        return await this.createWorkflow(payload);
+      
+      case 'start':
+        return await this.startWorkflow(payload.workflowId);
+      
+      case 'pause':
+        const workflow = this.workflows.get(payload.workflowId);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)